lib/travis/become/app.rb

```diff
@@ -29,12 +29,24 @@
   body 'route / not found, you most likely want /{login}'
 end
 
+
+route :get, :post, '/id/:login' do
+  handle_login(params, :by_id)
+end
+
 route :get, :post, '/:login' do
-  login = params['login']
+  handle_login(params, :by_login)
+end
 
+def handle_login(params, type)
+  login = params['login']
   begin
-    login_column = Travis::Become::User.arel_table[:login]
-    user = Travis::Become::User.where(login_column.lower.eq(login.downcase)).first
+    if type == :by_id
+      user = Travis::Become::User.find_by(id:  login)
+    else
+      login_column = Travis::Become::User.arel_table[:login]
+      user = Travis::Become::User.where(login_column.lower.eq(login.downcase)).first
+    end
```

Comment:
Just for a bit better clarity, I'd like to suggest using `:id` parameter for id, instead of it pretending `:login`:

```ruby
route :get, :post, '/id/:id' do
  handle_login(params, :by_id)
end

route :get, :post, '/:login' do
  handle_login(params, :by_login)
end

def handle_login(params, type)
  begin
    if type == :by_id
      user = Travis::Become::User.find_by(id:  params['id'])
    else
      login_column = Travis::Become::User.arel_table[:login]
      user = Travis::Become::User.where(login_column.lower.eq(params['login'].downcase)).first
    end
```
How does it sound to you?

Reply:
was done to have some common param from all routes, indeed 'login' is unfortunate and unnecessary, updated